src/react-apps/applications/service-development/src/config/routes.tsx

```diff
@@ -59,12 +59,12 @@ export const routes = [
     },
   },
   {
-    path: '/testingintestenvironment',
+    path: '/deploytotest',
     exact: true,
     activeSubHeaderSelection: 'Teste',
     activeLeftMenuSelection: 'Test i testmiljø',
     menu: 'test',
-    subapp: TestingInTestenvironmentContainer,
+    subapp: DeployToTestContainer,
```

Comment:
Are we really going to use the term container her?
Why not test environment?

Reply:
Renamed the import to DeployToTestEnvironment